libwild/src/args.rs

```diff
@@ -1090,6 +714,1414 @@ fn warn_unsupported(opt: &str) -> Result {
+        .execute(|_args, modifier_stack| {
+            modifier_stack.last_mut().unwrap().archive_semantics = false;
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("no-fork")
+        .help("Do not fork")
+        .execute(|args, _modifier_stack| {
+            args.should_fork = false;
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("update-in-place")
+        .help("Update file in place")
+        .execute(|args, _modifier_stack| {
+            args.file_write_mode = Some(FileWriteMode::UpdateInPlace);
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("EB")
+        .help("Big-endian (not supported)")
+        .execute(|_args, _modifier_stack| {
+            bail!("Big-endian target is not supported");
+        });
+
+    parser
+        .declare()
+        .long("prepopulate-maps")
+        .help("Prepopulate maps")
+        .execute(|args, _modifier_stack| {
+            args.prepopulate_maps = true;
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("verbose-gc-stats")
+        .help("Verbose GC statistics")
+        .execute(|args, _modifier_stack| {
+            args.verbose_gc_stats = true;
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("allow-shlib-undefined")
+        .help("Allow undefined symbols in shared libraries")
+        .execute(|args, _modifier_stack| {
+            args.allow_shlib_undefined = true;
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("no-allow-shlib-undefined")
+        .help("Disallow undefined symbols in shared libraries")
+        .execute(|args, _modifier_stack| {
+            args.allow_shlib_undefined = false;
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("error-unresolved-symbols")
+        .help("Error on unresolved symbols")
+        .execute(|args, _modifier_stack| {
+            args.error_unresolved_symbols = true;
+            Ok(())
+        });
+
+    parser
+        .declare()
+        .long("warn-unresolved-symbols")
+        .help("Warn on unresolved symbols")
+        .execute(|args, _modifier_stack| {
+            args.error_unresolved_symbols = false;
+            Ok(())
+        });
+
+    add_silently_ignored_flags(&mut parser);
+    add_ignored_flags(&mut parser);
+    add_default_flags(&mut parser);
+
+    parser
+}
+
+fn add_silently_ignored_flags(parser: &mut ArgumentParser) {
+    for flag in SILENTLY_IGNORED_FLAGS {
+        let mut declaration = parser.declare();
+        declaration = declaration.long(flag);
+        declaration.execute(|_args, _modifier_stack| Ok(()));
+    }
+}
+
+fn add_ignored_flags(parser: &mut ArgumentParser) {
+    for flag in IGNORED_FLAGS {
+        let mut declaration = parser.declare();
+        declaration = declaration.long(flag);
+        declaration.execute(|_args, _modifier_stack| Ok(()));
```

Comment:
Previously, flags listed in `IGNORED_FLAGS` issued a warning. However, now it looks like they're treated the same as `SILENTLY_IGNORED_FLAGS`.